gulpfile.babel.ts

```diff
@@ -175,108 +125,131 @@ async function browserifyTask(options: BrowserifyTaskOptions): Promise<void> {
+    module: {
+      rules: [
+        {
+          exclude: /node_modules/,
+          test: /\.m?[jt]sx?$/,
+          use: {
+            loader: 'babel-loader',
+            options: {
+              plugins: [
+                [
+                  'transform-inline-environment-variables',
+                  {
+                    include: ['NODE_ENV', 'IMPLEMENTATION_URL', 'VERSION'],
+                  },
+                ],
+              ],
+            },
+          },
+        },
+        {
+          test: /\.css$/,
+          use: [
+            {
+              loader: 'style-loader',
+              options: {
+                insert: (
+                  htmlElement: HTMLElement,
+                  _options: Record<string, any>
+                ) => {
+                  if (!document.getElementById('inboxsdk__style')) {
+                    htmlElement.id = 'inboxsdk__style';
+                    if (!document.head) throw new Error('missing head');
+                    document.head.appendChild(htmlElement);
+                  }
+                },
+              },
+            },
             {
-              include: ['NODE_ENV', 'IMPLEMENTATION_URL', 'VERSION'],
+              loader: 'css-loader',
+              options: {
+                // importLoaders: 1,
+                modules: {
+                  exportLocalsConvention: 'dashesOnly',
+                  localIdentName: args.production
+                    ? 'inboxsdk__[hash:base64]'
+                    : 'inboxsdk__[name]__[local][hash:base64]',
+                  // This keeps the spirit of local hash names changing every hour that used to be in idMap.js
+                  localIdentHashSalt:
+                    '' + Math.floor(Date.now() / (1_000 * 60 * 60)),
+                  mode: (resourcePath: string) =>
+                    resourcePath.endsWith('.module.css') ? 'local' : 'global',
+                  namedExport: true,
+                },
+              },
             },
           ],
-        ],
+        },
+      ],
+    },
+    optimization: {
+      minimize: willMinify,
+    },
+    output: {
+      path: path.join(
+        __dirname,
+        options.writeToPackagesCore ? 'packages/core/' : 'dist'
+      ),
+      filename: destName,
+      ...(options.standalone
+        ? {
+            libraryTarget: 'umd',
+            libraryExport: 'default',
+            library: options.standalone,
+          }
+        : {}),
+    },
+    plugins: [
+      // Work around for Buffer is undefined:
+      // https://github.com/webpack/changelog-v5/issues/10
+      new webpack.ProvidePlugin({
+        Buffer: ['buffer', 'Buffer'],
+        // https://stackoverflow.com/a/64553486 for default-encoding
+        // https://github.com/algolia/places/issues/847#issuecomment-748202652 For Algolia
+        process: 'process/browser',
       }),
-      { extensions: ['.js', '.tsx', '.ts'] }
-    )
-    .transform('redirectify', { global: true });
```

Comment:
The `lodash/once` overwrite we were doing previously looked like it was shaving ~1KB off output size. There may have been a perf rationale for it that's been lost to time though. @Macil, if you remember, you might weigh in.